src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -337,8 +337,8 @@ public function testOverwriteWithSectionOutput()
         rewind($output->getStream());
         $this->assertEquals(
             '  0/50 [>---------------------------]   0%'.\PHP_EOL.
-            "\x1b[1A\x1b[0J".'  1/50 [>---------------------------]   2%'.\PHP_EOL.
-            "\x1b[1A\x1b[0J".'  2/50 [=>--------------------------]   4%'.\PHP_EOL,
+            "\x1b[2A\x1b[0J".'  1/50 [>---------------------------]   2%'.\PHP_EOL.
```

Comment:
This makes sense. 2 lines are cleared each time: 1 for the progress bar and 1 for the extra new line on the end.